src/shims/ApiCompatBaseline.netcoreapp.netstandard20.txt

```diff
@@ -1,46 +1,75 @@
+Compat issues with assembly Microsoft.VisualBasic:
+TypesMustExist : Type 'Microsoft.VisualBasic.CallType' does not exist in the implementation but it does exist in the contract.
```

Comment:
These look wrong.  I'd expect an MS.VB facade to be produced that forwards to MS.VB.Core